app/cdap/components/ParsingDrawer/Components/ParsingHeaderActionTemplate/index.tsx

```diff
@@ -60,10 +58,10 @@ export default function(props) {
           src="/cdap_assets/img/import.svg"
```

Comment:
if import icon is there in MUI icons use them
if we don't have use it as a component

Reply:
done